### ISSUE TITLE
Add vendor: Retool

### ISSUE THREAD

**Issue by nicotriballier** [ssotax/ssotax#12]
```yaml
---
base_pricing: 5
currency: USD
free_sso_providers:
  - Google
name: Retool
notes: minimum $60k/y + ~$23.75/end user/month to switch to Enterprise plan that unlocks SSO. Retool also capped their business plan to 25 users forcing essentially every non-startup to go Enterprise
pricing_scheme: fixed cost per year (for Enterprise) + variable cost per end user/month (all plans)
pricing_sources:
  - https://retool.com/pricing
  - Quote
scim_pricing: 60000
sso_pricing: 60000
updated_at: 2023-12-30
vendor_url: https://retool.com
```

**Comment by MMore** [comment]
Hey @nicotriballier , thanks for your contribution.
To calculate the %Increase properly, we need to be a) as close to a base pricing as possible, b) stay in a pricing scheme (base pricing does not fit the sso pricing). 
I can see that there is also a standard user, not just an end user. Can we integrate that into the base pricing? Also the fixed fee you got in that quote _could_ be dependent on the number of user you have in your orgs. Maybe it makes sense to take that into consideration. For instance we could use the number of users you have and calculate a price per user out of it so that it compares better with other vendors (exemplary). What do you think?

**Comment by nicotriballier** [comment]
Hi @MMore , noted thx. Here's a revised version:
---
base_pricing: 5
currency: USD
free_sso_providers:
  - Google
name: Retool
notes: Enterprise has a $60k/y fixed cost + ~$23.75/end user/month variable cost. Retool also capped their on-prem business plan to 25 users forcing essentially every non-startup to go Enterprise. $216.06/end user/month = $60k/12/26 + $23.75 (for a team of 26 end users)
pricing_scheme: price 'end user'/month 
pricing_sources:
  - https://retool.com/pricing
  - Quote
scim_pricing: 216.06
sso_pricing: 216.06
updated_at: 2024-01-01
vendor_url: https://retool.com

**Comment by MMore** [comment]
Thank you @nicotriballier !
